sass/elements/_tables.scss

```diff
@@ -2,3 +2,108 @@ table {
 	margin: 0 0 1.5em;
 	width: 100%;
 }
+
+#tix {
```

Comment:
Let's move all the tickets related code out to `sass/site/primary/`, since that's where the other page-specific styles are.